DataProc/MergeOutputs/merge.C

```diff
@@ -1,11 +1,14 @@
-void merge(const char *filesToMerge="FilterEvents_Trees.root")
+void merge(const char *filesToMerge="FilterEvents_Trees.root", const char *inputCollection="wn.xml")
```

Comment:
This change introduces an argument to the merge function, allowing to specify a different XML collection. Since Grid JDLs don't use the second parameter, they will always fall back to the default one, `wn.xml`. This part _does_ affect Grid jobs but it was tested to work.